AISKU/Tests/Unit/src/ThrottleSentMessage.tests.ts

```diff
@@ -0,0 +1,244 @@
+import { ApplicationInsights, ApplicationInsightsContainer, IApplicationInsights, IConfig, IConfiguration, LoggingSeverity, Snippet, _eInternalMessageId } from '../../../src/applicationinsights-web'
+import { AITestClass, Assert} from '@microsoft/ai-test-framework';
+import { IThrottleInterval, IThrottleLimit, IThrottleMgrConfig } from '@microsoft/applicationinsights-common';
+import { SinonSpy } from 'sinon';
+import { AppInsightsSku } from '../../../src/AISku';
+import { createSnippetV5 } from './testSnippetV5';
+import { FeatureOptInMode, newId } from '@microsoft/applicationinsights-core-js';
+import { createSnippetV6 } from './testSnippetV6';
+
+const TestInstrumentationKey = 'b7170927-2d1c-44f1-acec-59f4e1751c11';
+
+export class ThrottleSentMessage extends AITestClass {
+    private _ai: IApplicationInsights;
+    private getAi: ApplicationInsights;
+    private _config: IConfiguration | IConfig;
+    private _logger;
+
+    constructor() {
+        super("ThrottleSentMessage");
+    }
+
+    public _getTestConfig() {
+        let tconfig = {
+            disabled: false,
+            limit: {
+                samplingRate: 1000000,
+                maxSendNumber:100
+            } as IThrottleLimit,
+            interval: {
+                monthInterval: 1,
+                dayInterval: undefined
+            } as IThrottleInterval
+        } as IThrottleMgrConfig;
+        let config: IConfiguration | IConfig = {
+            instrumentationKey: TestInstrumentationKey,
+            disableAjaxTracking: false,
+            disableFetchTracking: false,
+            enableRequestHeaderTracking: true,
+            enableResponseHeaderTracking: true,
+            maxBatchInterval: 2500,
+            disableExceptionTracking: false,
+            enableCorsCorrelation: true,
+            samplingPercentage: 50,
+            convertUndefined: "test-value",
+            disablePageUnloadEvents: [ "beforeunload" ],
+            throttleMgrCfg: {
+                [_eInternalMessageId.InstrumentationKeyDeprecation]:tconfig,
+                [_eInternalMessageId.SnippetUpdate]:tconfig,
+                [_eInternalMessageId.CdnDeprecation]:tconfig
+            },
+            featureOptIn:{
+                ["disableIkeyDeprecationMessage"]: {mode: FeatureOptInMode.enable},
+                ["disableCdnDeprecationMessage"]: {mode: FeatureOptInMode.enable},
+                ["disableSnippetVersionUpdateMessage"]: {mode: FeatureOptInMode.enable}
```

Comment:
As we have changes the public name to SdkLoader we should update these new names to reflect.